src/fragmentarium/ui/display/Transliteration.tsx

```diff
@@ -143,16 +151,60 @@ function DisplayLine({
   )
 }
 
+function DollarLineAndAtLine({
+  line,
+  container = 'div'
+}: {
+  line: LineContainer
+  container?: string
+}): JSX.Element {
+  if (line.type === 'RulingDollarLine') {
+    const rulingDollarLine = line as RulingDollarLine
+    return React.createElement(
+      container,
+      { className: classNames([`Transliteration__ControlLine`]) },
+      [
+        <hr
+          key={101}
+          className={`Transliteration__${line.type} ${rulingDollarLine.number}`}
+        />
+      ]
+    )
+  } else {
+    return React.createElement(
+      container,
+      { className: classNames([`Transliteration__ControlLine`]) },
+      [
+        <div key={101} className="Transliteration__DollarLineAndAtLine">
```

Comment:
temporary solution to pass a random prop.

Reply:
That is not a random key. Its a constant.

Reply:
Why not just pass a single child?

Why have to elements with element class? Use the correct element class above and just use plain string or span here.